src/base/bittorrent/session.cpp

```diff
@@ -2197,7 +2194,8 @@ void Session::saveResumeData()
     while (m_numResumeData > 0) {
         const std::vector<lt::alert *> alerts = getPendingAlerts(lt::seconds(30));
         if (alerts.empty()) {
-            fprintf(stderr, " aborting with %d outstanding torrents to save resume data for\n", m_numResumeData);
+            LogMsg(tr("Error: Abort saving resume data for %1 outstanding torrents.").arg(QString::number(m_numResumeData))
```

Comment:
@Chocobo1:  I would write it as "**Aborted** saving resume data" (past tense), for consistency with other log messages.

Reply:
Thanks! and updated.